interface/ceed-vector.c

```diff
@@ -357,30 +357,34 @@ int CeedVectorSetValue(CeedVector vec, CeedScalar value) {
   Note: The `CeedVector` must already have valid data set via @ref CeedVectorSetArray() or similar.
 
   @param[in,out] vec   `CeedVector`
-  @param[in]     start First index to set
+  @param[in]     start First index to set in range `[start, stop)`
+  @param[in]     stop  Last index to set in range `[start, stop)`, or `-1` for `length`
```

Comment:
I think the wording is ambiguous and readers may not catch the subtle half-open interval.
```suggestion
  @param[in]     stop  Upper bound on range `[start, stop)`, or `-1` for `length`
```
or "one past the last element in range" (the C standard often uses such "one past the last element of an array object").